scaling_performance/optimizing_storage.adoc

```diff
@@ -314,32 +312,65 @@ loopback mounted sparse file. To see if your system is using the loopback mode:
 
 [IMPORTANT]
 ====
-Red Hat strongly recommends using the `DeviceMapper` storage driver in thin-pool mode for production workloads.
+Red Hat strongly recommends using the DeviceMapper storage driver in thin-pool mode for production workloads.
 ====
 
-`Overlay` is also supported for container runtimes use cases as of Red Hat Enterprise Linux
+OverlayFS is also supported for container runtimes use cases as of Red Hat Enterprise Linux
 7.2, and provides faster start up time and page cache sharing, which can
 potentially improve density by reducing overall memory utilization.
 
 [[benefits-of-using-the-overlay-graph-driver]]
-== Benefits of Using the Overlay Graph Driver with SELinux
-
-The default Docker storage configuration on Red Hat Enterprise Linux continues
-to be `DeviceMapper`. While the use of `Overlay` as the container's storage
-technology is under evaluation, moving Red Hat Enterprise Linux to `Overlay` as
-the default in future releases is under consideration. As of Red Hat Enterprise
-Linux 7.2, `Overlay` became a supported graph driver. As of Red Hat Enterprise
-Linux 7.4, SELinux and the `Overlay2` graph driver became a supported
-combination.
-
-The main advantage of the `Overlay` file system is Linux page cache sharing among
-containers sharing an image on the same node. This attribute of `Overlay` leads to
+=== Comparing the overlay vs. overlay2 Graph Drivers 
+
+OverlayFS is a type of union file system. It allows the user to overlay one file system on top of another. 
+Changes are recorded in the upper file system, while the lower file system remains unmodified. 
+This allows multiple users to share a file-system image, such as a container or a DVD-ROM, where the base image is on read-only media. 
+
+OverlayFS layers two directories on a single Linux host and presents them as a single directory. These directories are called layers 
+and the unification process is referred to as a union mount. 
```

Comment:
I think that the first paragraphs in this section should move to the previous section or that you should combine this section and the following section under a general Overlay heading.

Reply:
@kalexand-rh This section was already present in the docs. Not sure why it shows I added it. I swapped the two sections. Now it is essentially:
Choosing DeviceMapper or OverlayFS
Why choose OverlayFS if you use SELinux
Deciding which graph driver to use (overlay or overlay2) if you choose OverlayFS

Reply:
That's fair. I think that the heading makes it sound like the information should be focused on the differences between the versions and that the overall description might make more sense in the previous section, but if it's legacy, don't worry with it.